kuma/static/img/subscriptions/credit-card.svg

```diff
@@ -0,0 +1,3 @@
+<svg class="icon icon-credit-card" xmlns="http://www.w3.org/2000/svg" width="21" height="16" aria-hidden="true">
```

Comment:
We should remove the `width` and `height` and replace with `viewBox` for responsiveness(especially in IE11)

Reply:
```
<svg class="icon icon-credit-card" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 21 16" aria-hidden="true" role="img"><path d="M18.85 0H1.72C.77 0 0 .77 0 1.71V14.3c0 .93.77 1.7 1.72 1.7h17.13c.95 0 1.72-.77 1.72-1.71V1.7c0-.93-.77-1.7-1.72-1.7zM1.93 1.71h16.7c.13 0 .22.1.22.22v1.5H1.72v-1.5c0-.12.1-.22.21-.22zm16.7 12.58H1.94a.21.21 0 01-.21-.22V8h17.13v6.07c0 .12-.1.22-.21.22zM6.87 10.7v1.43c0 .24-.2.43-.43.43H3.86a.43.43 0 01-.43-.43V10.7c0-.23.2-.42.43-.42h2.57c.23 0 .43.19.43.42zm6.85 0v1.43c0 .24-.19.43-.42.43H8.43a.43.43 0 01-.43-.43V10.7c0-.23.2-.42.43-.42h4.86c.23 0 .42.19.42.42z"/></svg>
```